pkg/products/observability/reconciler.go

```diff
@@ -148,10 +158,6 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
-func (r *Reconciler) reconcileSecrets(_ context.Context, _ k8sclient.Client, _ *integreatlyv1alpha1.RHMI) (integreatlyv1alpha1.StatusPhase, error) {
```

Comment:
Is this removed because observability will create it?

Reply:
This function was a placeholder in the install and is replaced with monitoringcommon.ReconcileAlertManagerSecrets https://github.com/valerymo/integreatly-operator/pull/7/files#diff-714b16293aa3138fdbfef49420c8aeecccf5686379636d63c174d0f4c7089fb9R265